scripts/e2e

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+if [ "${ARCH}" != "amd64" ]; then
+    exit 0
+fi
+
+cd $(dirname $0)/..
+
+if [ ! -f ./bin/terraform-controller ]; then
+    echo "./bin/terraform-controller not built, make build-controller to create."
+    exit 1
+fi
+
+k3s server&
+k3s_pid=$!
```

Comment:
Why did the cleanup for the processes get dropped? If I run this script lots of things are going to leak for me locally

Reply:
long convo and tldr; we will do this next tie.